contracts/monitoring/src/msg.rs

```diff
@@ -15,11 +15,14 @@ pub enum ExecuteMsg {
         chain_name: ChainName,
         new_prover_addr: Addr,
     },
+    RegisterActiveWorkerSet {
+        next_worker_set: WorkerSet,
+    },
 }
 
 #[cw_serde]
 #[derive(QueryResponses)]
 pub enum QueryMsg {
     #[returns(WorkerSet)]
-    GetActiveVerifiersForChain { chain: ChainName },
+    GetActiveWorkerSet { prover_address: Addr },
```

Comment:
A couple of things here:

1. I think it makes more sense to just pass the chain name, instead of the prover address. The caller probably doesn't know the prover address (if they did, they could just query the prover directly). One of the main jobs of this contract is to keep track of the prover address.
2. If this query is not actually going to be used by the service registry or some other contract, I would just remove it.

Reply:
1. Sure
2. It is going to be used. For now, we know for certain axelarscan is going to need it.